### DISCUSSION TITLE
Visão geral Razor Pages : Minhas configurações e ViewImports não estão sendo repassadas para as views

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#661] [Tira Dúvidas] [answered]
Escrevi o namespace para para as views no ViewImports, mas ele ficando dando problemas dizendo que não encontra os dados do model. Só que quando preencho o caminho inteiro do model ```@model RazorApp.Pages.Index``` aí funciona, só que não era para estar funcionando?

Obs: não é uma questão de recompilar pois eu já parei e refiz isso algumas vezes.

![image](https://github.com/user-attachments/assets/6f5621d3-e912-43a9-aba0-b038f7793527)

Dessa forma funciona, mas aí eu estou repetindo em 2 lugares.
![image](https://github.com/user-attachments/assets/6f2d7dba-70c5-4af3-b451-ac95074b79b3)

**Comment by oimarcella** [comment]
Vi que faltava adicionar um using RazorApp:

```
@using RazorApp
@namespace RazorApp.Pages
@using System.Globalization

@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
```
Porém ainda sim, ele ainda acusa o mesmo erro.

**Reply by andrebaltieri** [reply]
@oimarcella parabéns pelo progresso 🚀
Quero ver você nos próximos desafios!

**Reply by oimarcella** [reply, marked as answer]
> Porém ainda sim, ele ainda acusa o mesmo erro.

Então, realmente parece que a solução era oque estava no meu próprio comentário acima, porém na página Index, realmente não consigo usar somente o nome, preciso passar o caminho todo. Nas outras páginas deu para usar somente o último nome.

Pra vocês verem:
![image](https://github.com/user-attachments/assets/cdef0f47-88cd-41f0-880e-669363832081)

**Reply by andrebaltieri** [reply]
Posta o código do seu `_ViewImports.cshtml` por favor!

**Reply by oimarcella** [reply]
![image](https://github.com/user-attachments/assets/5ee8e479-c854-4d10-b85e-5f201992d304)
Porém como disse, em outras páginas funcionou, só tive que colocar o caminho completo para  a Index.
